src/__tests__/acceptance/user.controller.acceptance.ts

```diff
@@ -28,15 +20,5 @@ describe("UserController (acceptance)", () => {
         .send(credentials)
         .expect(401);
     });
-
-    it("should resolve in a jwt token when logging in with the correct credentials", async () => {
```

Comment:
Question. Why was this test removed?

Reply:
After refactoring JWT validation is no longer needed so is for the JWT test. 
Within this PR synapse credential will be directly passed to loopback from SciCat BE, then loopback will use the credential to login Synapse server and save generated token in the memory.